src/ui/pages/SwapForm/SlippageSettings/SlippageSettings.tsx

```diff
@@ -91,6 +95,67 @@ function SlippageWarning({ percentValue }: { percentValue: string }) {
   );
 }
 
+function PercentWidth({ onValue }: { onValue(width: number): void }) {
+  const persentCharWidthRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    let interval: ReturnType<typeof setTimeout>;
+    if (persentCharWidthRef.current) {
+      onValue(persentCharWidthRef.current.clientWidth);
+    } else {
+      interval = setTimeout(
+        () => onValue(persentCharWidthRef.current?.clientWidth || 0),
+        100
+      );
+    }
+    return () => clearTimeout(interval);
+  }, [onValue]);
+
+  return (
+    <UIText
+      ref={persentCharWidthRef}
+      kind="body/accent"
+      style={{ visibility: 'hidden', position: 'absolute' }}
```

Comment:
Btw, we should either unmount element from DOM after measuring or apply some styles or attributes so that it is ignored by screenreaders

Reply:
Will add an attribute

Reply:
Ah, wait, `visibility: hidden` already does that
Still may be a good idea to unmount